src/components/Link.tsx

```diff
@@ -106,10 +114,32 @@ export const Link = (props: LinkProps) => {
   }
 
   if (isExternal) {
+    let href = '';
+    if (isEthereumDotOrg) {
+      // Grab everything after "ethereum.org/"
+      const slug: string = to.substr(
+        to.indexOf(ETHEREUM_DOT_ORG) + ETHEREUM_DOT_ORG.length + 1
```

Comment:
If we just check for `const ETHEREUM_DOT_ORG = 'https://ethereum.org'`, then `to.indexOf(ETHEREUM_DOT_ORG)` will always be 0.